.github/workflows/ci.yml

```
@@ -181,6 +181,38 @@ jobs:
       - name: Setup Nix cache
         uses: DeterminateSystems/magic-nix-cache-action@908b263ff629f4cc17666315b7fd3ec127c6244d # v14
 
+      - name: Test downstream overlay composition
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:1:26: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)